### ISSUE TITLE
Code quality (`UploadActivity`): Use `DialogUtil` method instead of re-implementing in `showInfoAlert`

### ISSUE THREAD

**Issue by chan-j-d** [commons-app/apps-android-commons#5148]
In `UploadActivity`, there is a method 
https://github.com/commons-app/apps-android-commons/blob/c88ce144e15f7a2e9340042b254b2babe98489c5/app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java#L460-L471

implemented and used only once to display a dialog alert. I think it might be better to reuse `DialogUtil::showAlertDialog` to not duplicate the code required to display an alert dialog.

**Comment by chan-j-d** [comment]
@nicolas-raoul if it's fine with you, I'll add a PR for this. Thanks!

**Comment by nicolas-raoul** [comment]
Yes thanks! 🙂

**Comment by chan-j-d** [comment]
From looking through usages of the builder, I realized that there are many instances of classes using `AlertDialog.Builder` where its usage can be covered by one of the in-house `DialogUtil` methods that might be more straightforward to use. This is likely due to `DialogUtil` being a utility class added much later in the project's life. I was wondering if it might be appropriate to refactor all/most of these to use `DialogUtil.showAlertDialog` methods

**Comment by nicolas-raoul** [comment]
> I was wondering if it might be appropriate to refactor all/most of these to use DialogUtil.showAlertDialog methods

Yes please! :-) Just create a new issue about it and feel free to start working on it. A single pull request for all is probably enough, but feel free to split into several pull requests if you prefer.
